atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/api/watch/LockWatchEventCacheIntegrationTest.java

```diff
@@ -238,24 +238,22 @@ public void cacheClearedOnSnapshotUpdate() {
     }
 
     @Test
-    public void getEventsForTransactionsReturnsSnapshotWithOldEvents() {
+    public void getEventsForTransactionsReturnsSnapshotWithCondensedEvents() {
```

Comment:
this test keeps having its behaviour changed, but it didn't make a huge amount of sense with the new scheme to leave as-is.